### DISCUSSION TITLE
Release 1.8.1

### DISCUSSION THREAD

**Discussion by whalley** [moneymanagerex/moneymanagerex#6866] [Development]
@guanlisheng  @n-stein @vomikan Have pushed all open issues planned for 1.8.1 out to 1.9.0 as nothing looked too major. 

Will issue an RC build for final testing. We can look to release by end of the month.  Suggest **not** merging any non-essential changes until 1.8.1 is released.

**Comment by vomikan** [comment]
https://github.com/moneymanagerex/moneymanagerex/pulls

These two pull requests look quite competent. I prefer to use all contributors' efforts in our project.
Should we put this on hold or include it in the next release?

**Reply by whalley** [reply]
General thought was that we hold off this change as it still requires a few tasks completing and testing. Idea was to get 1.8.1 out very soon then incorporate these other changes in a 1.9.0 release by end of year.

**Comment by whalley** [comment]
Are we good the release 1.8.1 of Friday this week? Only outstanding issue I see is https://github.com/moneymanagerex/moneymanagerex/issues/6863 but this seems isolated to that particular user/databse and suggest we move to 1.9.0.

Views?

**Comment by renato-mmex** [comment]
I expected version 1.8.1 to be released yesterday.
Do you have a new date?

**Reply by whalley** [reply]
https://github.com/moneymanagerex/moneymanagerex/issues/6847 needs fixing first.

**Comment by renato-mmex** [comment]
Apparently there are problems with the realisation of #6847 .
I would suggest moving this fix to 1.9.0 and releasing 1.8.1.

**Reply by guanlisheng** [reply]
i second Rene's suggestion and hope we can roll out v1.8.1 earlier.

**Comment by n-stein** [comment]
I don't think there is any open issue with #6847. The assertion errors are related to the wxDate:ToUTC call in mmDateRange not including a time zone. These only show up if you build wxWidgets with debug enabled. If you run release wxWidgets there are no errors which is why beta testers running the RC build have not reported it.

I will look at it tomorrow.

**Reply by n-stein** [reply]
fixed via PR #6933

**Comment by renato-mmex** [comment]
there are new findings ??

**Comment by georgeef** [comment]
New findings may never end. Since 1.8.1 is already delayed, I propose a simple rule: issues that existed also in 1.8.0 should be shifted after 1.8.1. This release may not be perfect and it does not solve all issues, but as long as it is an improvement with respect to the previous release, it makes sense for the users to have it earlier rather than later.

**Comment by whalley** [comment]
Think we are primarily awaiting confirmation that there are no issues with major https://github.com/moneymanagerex/moneymanagerex/issues/6847 on Windows...

**Comment by whalley** [comment]
I'm going to release v1.8.1 tomorrow unless anybody has objections....

**Comment by whalley** [comment]
v1.8.1 is released, @joshua-stone could you please ensure Flathub is updated.

@n-stein There seems an issue with the workflows for the Ubuntu builds which I think you created.
https://github.com/moneymanagerex/moneymanagerex/actions/runs/12087002620/job/33707529924

**Comment by federicospadaro** [comment]
Hi to all, 
after update this morning to the latest v.1.8.1, If I truing to open the database doesn't accept my password.
Till the last week, on friday all works fine.
I have restored the previous version the issue is solved

**Reply by renato-mmex** [reply]
https://github.com/moneymanagerex/moneymanagerex/issues/6971
